pkg/apiserver/resthandler.go

```diff
@@ -736,6 +737,10 @@ func setListSelfLink(obj runtime.Object, req *restful.Request, namer ScopeNamer)
 }
 
 func getPatchedJS(contentType string, originalJS, patchJS []byte, obj runtime.Object) ([]byte, error) {
+	// Remove "; charset=" if included in header.
```

Comment:
I think if anything a TODO would be fine. Do you want me to add one before it gets merged, or leave it for now?